xbmc/addons/interfaces/GUI/ListItem.cpp

```diff
@@ -34,12 +34,39 @@ extern "C"
 namespace ADDON
 {
 
+/// TEMPORARY PRESENT FOT TEST PURPOSE ONLY !!!
+static int fileitemcnt = 0;
+
+class CAddonFileItem : public CFileItem
+{
+public:
+  CAddonFileItem()
+  {
+    fprintf(stderr, "file item count: %i; %s\n", ++fileitemcnt, __PRETTY_FUNCTION__);
```

Comment:
`__PRETTY_FUNCTION__` can't be used on windows, replace it with `__FUNCTION__`
same is true for the other occurrences of `__PRETTY_FUNCTION__`